### ISSUE TITLE
Loosen restrictions on Scalatags package

### ISSUE THREAD

**Issue by matthughes** [japgolly/scalajs-react#61]
I've run into a couple missing HTML attributes with 0.7.0.

I tried to declare my own attributes:

```
  final val ariaLabel = "aria-label".attr
  final val ariaHidden = "aria-hidden".attr
```

but 'attr' is hidden in the Scalatags object which is private to vdom.  Would you consider making that public so people can build at least temporary shims?  Ideally, anything missing from HTML proper should get pushed upstream into this project, but I don't want to be stalled and unable to use a particular attribute/tag until a PR gets pushed and released.

**Comment by japgolly** [comment]
@matthughes This and the className issue fixed and just released in 0.7.1.

```scala
  final val ariaLabel = "aria-label".reactAttr
  final val ariaHidden = "aria-hidden".reactAttr
```

**Comment by matthughes** [comment]
Awesome, thanks.
